src/embezzler.ts

```diff
@@ -49,44 +51,56 @@ import { Macro, shouldRedigitize, withMacro } from "./combat";
 import { usingThumbRing } from "./dropsgear";
 import { crateStrategy, equipOrbIfDesired } from "./extrovermectin";
 import { baseMeat, globalOptions, ltbRun, WISH_VALUE } from "./lib";
-import { determineDraggableZoneAndEnsureAccess, draggableFight } from "./wanderer";
+import { familiarWaterBreathingEquipment, waterBreathingEquipment } from "./outfit";
+import { determineDraggableZoneAndEnsureAccess, DraggableFight } from "./wanderer";
 
 /**
  * Configure the behavior of the fights in use in different parts of the fight engine
  * @interface EmbezzlerFightConfigOptions
  * @member {Requirement[]?} requirements maximizer requirements to use for this fight (defaults to empty)
  * @member {draggableFight?} draggable if this fight can be pulled into another zone and what kind of draggable it is (defaults to undefined)
- * @member {boolean?} canStartWanderers if this fight can be used to initialize wanderers (defaults to true)
+ * @member {boolean?} canInitializeWandererCounters if this fight can be used to initialize wanderers (defaults to true)
```

Comment:
I still think this docstring isn't quite accurate. This doesn't tell us whether the fight itself initializes the counter--spooky putty, for example, doesn't initialize it--this tells us if it's safe to start our wanderer counter by adventuring in another location when this fight is up. I think.